### PR TITLE
fix(jsx): boolean attribute

### PR DESCRIPTION
**Pull request by PatrickJS**
## Description
fix https://github.com/BuilderIO/mitosis/issues/732

### CODE REVIEW

File: packages/core/src/parsers/jsx/element-parser.ts [hunk 1]
```diff
@@ -190,6 +190,12 @@ export const jsxElementToJson = (
       if (types.isJSXAttribute(item)) {
         const key = item.name.name as string;
         const value = item.value;
+
+        // boolean attribute
+        if (!item.hasOwnProperty('value')) {
```
Comment (samijaber) [inline packages/core/src/parsers/jsx/element-parser.ts:195]: I think we can reuse `value` from above
```suggestion
        if (value === null) {
```

**Review by samijaber — COMMENTED** [review]
Probably need to `yarn snapupdate`

**Review by samijaber — APPROVED** [review]
Can we add one of these props to a snapshot to make sure it works

**Comment by PatrickJS** [comment]
I'm adding a snapshot because the preview deployment didn't show it fixed

**Comment by PatrickJS** [comment]
snaps show it works and even in the preview now

Screen Shot 2022-09-15 at 10 47 54 AM

**Pull request merged**